docs(README): remove `clone repository` from usage

uncenter opened the pull request:
I think these usage instructions more accurately represent the instructions for a real app, and they demonstrate some good practices: linking to the themes directory, using `**X** > **Y** > **Z**` for step by steps, etc. I also think commenting out the FAQ makes it more clear that that is the optional section that we are referring to, and makes it so maintainers don't feel "pressured" to add something random to fill it.

uncenter committed:
docs: add more realistic usage instructions and comment out FAQ section

uncenter committed:
docs: update steps once again

sgoudham commented on README.md:
```diff
@@ -36,16 +36,18 @@
-2. Open the app's settings
-3. Select `import theme` and browse to where you cloned Catppuccin
-4. Select it
+1. Download the flavor of your choice from [`themes/`](./themes/).
```
Don't think we should reference a non-existent directory

uncenter replied:
Yeah it was an optimistic followup of my Whiskers one. What should I use instead? Should we create a `themes/.gitkeep`?

sgoudham replied:
Yeah to be honest, my ideal diff is removing the `Clone this repository locally` step since most of our ports don't require the entire repository to be cloned.

 Since we expect them to delete it anyways, what about quite simple and vague like:

```
1. Download theme
2. Apply theme
3. Enjoy!
```

uncenter replied:
How does it look now?

uncenter committed:
docs: update usage steps

sgoudham committed:
docs(README): keep FAQ visible by default

I think it's nicer for the rendered template
on catppuccin/template to highlight all the
possible sections explicitly.

Ultimately the person will have to interact
with the section anyways so I don't think it
needs to be modified.

sgoudham reviewed (approved):
Kept the FAQ visible, see the commit for why.
Thanks for updating the instructions to avoid cloning the repository to be the default
